wzdx/combination/attenuator.py

```diff
@@ -0,0 +1,190 @@
+import json
+from datetime import datetime
+from wzdx.tools import cdot_geospatial_api, polygon_tools
+import logging
+
+ATTENUATOR_TIME_AHEAD_SECONDS = 30 * 60
```

Comment:
Are we planning on making this configurable as a passed in value? If not maybe we should name it "ATTENUATOR_THIRTY_MINUTES_AHEAD_SECONDS" so we know we're always talking about 30 minutes.

Reply:
I want this to be changeable in the future. it could be an environment variable?

Reply:
Environment variable would be a great option here. Did you want to add that with this PR or hold off?

Reply:
I think I want to hold off, will affect deployments